R/utils.R

```diff
@@ -0,0 +1,23 @@
+#' Read in a epidist Stan code chunk
+#'
+#' @param path The path within the "stan" folder of the installed epidist
+#' package to the Stan code chunk of interest.
+#' @return A character string containing the Stan code chunk of interest.
+#' @family utils
+#' @export
```

Comment:
For an issue. We might want this to be internal unless we want to support users loading in the stan code?

Reply:
If we see people making their own models then I see the arg for why it would be internal. However if we do see that it suggests to me the brms wrapping we do should be easier for them to reproduce vs having to copy and paste out of epidist

Reply:
Adding an issue for which functions to be internal versus external.